packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -29,20 +29,32 @@ export const pausingUploadReason = Symbol('pausing upload, not an actual error')
 class MultipartUploader {
```

Comment:
This 
```suggestion
/**
* A MultipartUploader instance is used per file upload to manage the chunk splitting,
* the S3 multipart upload phase: create, sign, upload, finish, and handling pause/resume.
* It also determines whether a upload should be done as multipart (`shouldUseMultipart`)
* or as a regular S3 upload.
*/
class MultipartUploader {
```

Reply:
The "S3 multipart upload phase" is not managed in this class.